src/pages/wallet/transaction.js

```diff
@@ -199,9 +200,12 @@ class Transaction extends Component {
   render() {
     const { navigation } = this.props;
     const {
-      transactionState, transactionId, amount, dateTime, memo, confirmations, title, stateIcon, isRefreshing, from, to,
+      transactionState, transactionId, amount, dateTime, memo, confirmations, title, stateIcon, isRefreshing, from, to, chain,
     } = this.state;
 
+    const fromChecksum = common.toChecksumAddressIfNeeded(from, chain);
```

Comment:
We only have the chain name here, would be best to get and send networkId for checksum format check